custom_components/plugwise_usb/models.py

```diff
@@ -0,0 +1,196 @@
+"""Models for the Plugwise USB integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntityDescription,
+)
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntityDescription
+from homeassistant.const import (
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    EntityCategory,
+    UnitOfEnergy,
+    UnitOfPower,
+    UnitOfTime,
+)
+from homeassistant.helpers.entity import EntityDescription
+
+from .const import STICK, USB_MOTION_ID, USB_RELAY_ID
+
+
+@dataclass
+class PlugwiseRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    plugwise_api: str
+
+
+@dataclass
+class PlugwiseEntityDescription(EntityDescription, PlugwiseRequiredKeysMixin):
```

Comment:
class PlugwiseEntityDescription(EntityDescription):

Reply:
Or remove this class and rename below classes to SensorEntityDescription, etc.

Maybe even better, move all descriptions (back) into the relevant files, as done in Core Plugwise.

Reply:
Tracking via https://github.com/plugwise/plugwise_usb-beta/issues/3